gcc_4.9-x86/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM i386/ubuntu:trusty
+FROM i386/ubuntu:xenial
```

Comment:
I need to keep 4.9 on trusty because of the issues with the glibc. I changed it some commits ago, I don't know if you remember:

https://github.com/conan-io/conan-docker-tools/commit/60c4f542929580f4bcdbb44b6564db105351c762

Reply:
understood.